pubsub/test/topics.test.js

```diff
@@ -96,7 +96,7 @@ describe('pubsub:topics', function () {
       var sample = getSample();
       var error = 'error';
       sample.mocks.topic.delete.callsArgWith(0, new Error(error));
-      sample.program.delete(topicName, function (err) {
+      sample.program.deleteTopic(topicName, function (err) {
```

Comment:
These renaming changes are necessary - but they don't seem to be related to `yargs`. :stuck_out_tongue: